package.json

```diff
@@ -163,13 +163,15 @@
     "throng": "^4.0.0",
     "tiny-queue": "^0.2.1",
     "twitter-text": "3.1.0",
+    "utf-8-validate": "^5.0.4",
```

Comment:
Do you need this dep?

Reply:
https://www.npmjs.com/package/ws#opt-in-for-performance-and-spec-compliance